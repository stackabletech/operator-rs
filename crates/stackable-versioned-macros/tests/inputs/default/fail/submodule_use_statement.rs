use stackable_versioned::versioned;

fn main() {
    #[versioned(version(name = "v1alpha1"))]
    mod versioned {
        mod v1alpha1 {
            struct Foo;
        }
    }
}
