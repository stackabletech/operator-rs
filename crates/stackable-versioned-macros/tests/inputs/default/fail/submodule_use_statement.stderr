error: submodules must only define `use` statements
 --> tests/inputs/default/fail/submodule_use_statement.rs:7:13
  |
7 |             struct Foo;
  |             ^^^^^^
